website/docs/tailwind/helpers.md

```diff
@@ -0,0 +1,150 @@
+---
+id: helpers
+title: Helpers
+---
+
+Frontend libs Tailwind includes many useful helpers.
+
+Here are a few featured:
+
+## General helpers
+
+### `getHiddenOptions`
+In order to simplify hiding parts of component options, the `getHiddenOptions` helper was created.
+
+To use, first in your component add:
+```jsx
+export const DemoOptions = (attributes) => {
+	const {
+		setAttributes,
+		// highlight-next-line
+		hideOptions,
+		additionalControls,
+		...rest
+	} = attributes;
+
+	...
+
+	// highlight-next-line
+	const hiddenOptions = getHiddenOptions(hideOptions);
+
+	...
+
+	return (
+		<>
+			<Toggle
+				label={__('My option', 'fe-libs-tailwind')}
```

Comment:
```suggestion
				label={__('My option', 'fe-libs-tailwind')}
```
Text domain has changed from `eightshift-frontend-libs`? I understand the `tailwind` addition but the change is significant here, can we have it somewhat similar... maybe I'm missing something here 😄

Reply:
This is a sample component, the textdomain will change based on your theme